modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/configuration/CPDefinitionLinkTypeSettingsImpl.java

```diff
@@ -5,46 +5,40 @@
 
 package com.liferay.commerce.product.internal.configuration;
 
+import com.liferay.commerce.product.configuration.CPDefinitionLinkTypeConfiguration;
 import com.liferay.commerce.product.configuration.CPDefinitionLinkTypeSettings;
-import com.liferay.osgi.service.tracker.collections.map.ServiceReferenceMapperFactory;
-import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMap;
-import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMapFactory;
-import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.configuration.metatype.bnd.util.ConfigurableUtil;
+
+import java.util.Map;
 
-import org.osgi.framework.BundleContext;
 import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Modified;
 
 /**
  * @author Alessio Antonio Rendina
  */
-@Component(service = CPDefinitionLinkTypeSettings.class)
+@Component(
+	configurationPid = "com.liferay.commerce.product.configuration.CPDefinitionLinkTypeConfiguration",
```

Comment:
Hey @Valmir-unicap, please type on terminal `git grep com.liferay.commerce.product.configuration.CPDefinitionLinkTypeConfiguration` and you will see some usages of this configuration. So, this commit is breaking the current behavior.